packages/cadl-vscode/language-configuration.json

```diff
@@ -1,23 +1,55 @@
 {
```

Comment:
Do we need to have this file checked in or can we generate it to dist/ like .tmLanguage?

Reply:
yeah was asking about that.